File: javascript/solutions/oddities.js
// Problem: https://open.kattis.com/problems/oddities

var length = readline();
var line;
var result = '';

while (line = readline()) {
  result += line + ' is ' + oddOrEven(parseInt(line)) + '\n';
}

function oddOrEven(number) {
  return number % 2 === 0 ? 'even' : 'odd';
}

print(result);

File: python3/solutions/oddities.py
# Problem: https://open.kattis.com/problems/oddities

########## GENERATE TEST CASES LOCALLY ##########

dummy_input = [
  '3',
  '10',
  '9',
  '-5'
]

def input():
  return dummy_input.pop(0)

################## CODE #####################

num_tc = int(input())

while num_tc > 0:
  number = input()
  print(number + ' is even' if int(number) % 2 == 0 else number + ' is odd')
  num_tc -= 1